### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by disa6302**
*Issue #, if available:*

*Description of changes:*

Release 1.3.0 This PR will not be merged. It is a placeholder for review

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Release tagged at:
<Commit:  <to-be-filled>
+
+Whats’s new:
+- ICE server stats added
+- RTP outbound and inbound stats added (open source contribution)
+- Signaling stats available
+- Clang check added to Travis
+- MBedTLS support added. Choice between OpenSSL and mbedTLS available
```
Comment (lherman-cs) [inline RELEASE.md:10]: I think we should postpone this to the next release since I still need to verify some key metrics.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Improvements:
+- ICE candidate priority calculation bug fixed improving the candidate nomination process
+- Improved SDP conformance with RFC to work with CRLF and LF
+- Fixed interruption of sending process due to momentary network unreachable errors (open source contribution)
```
Comment (sharadrj) [inline RELEASE.md:15]: We should add a line about fix for high CPU usage issues

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Producer C Changes: (Commit: e7d4868)
+- Build system improvements
+- README improvements on SDK versioning, development and release
+- Added non-BSD strnstr implementation
+- Fixed static libcurl build failure
+
+Known Issues:
+* Incorrect transceiver direction breaks SDP exchange in Firefox and iOS
+* Reduce memory allocations in sending audio/video packets
+* Get information about the selected candidates such as IP address, server source, type of candidate chosen and protocol used
```
Comment (sharadrj) [inline RELEASE.md:26]: Not sure what this line means. Are you saying we currently don't have a to surface these information. If so, please rephrase.

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Producer C Changes: (Commit: e7d4868)
+- Build system improvements
+- README improvements on SDK versioning, development and release
+- Added non-BSD strnstr implementation
+- Fixed static libcurl build failure
+
+Known Issues:
+* Incorrect transceiver direction breaks SDP exchange in Firefox and iOS
+* Reduce memory allocations in sending audio/video packets
```
Comment (sharadrj) [inline RELEASE.md:25]: Lets re-phrase it from customer POV.

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+Release 1.2.1 of WebRTC C SDK
+
+Release tagged at: <Commit:  <to-be-filled>
+
+Whats’s new:
+- ICE server stats added
+- RTP outbound and inbound stats added (open source contribution)
+- Signaling stats available
+- Clang check added to Travis
```
Comment (lherman-cs) [inline RELEASE.md:9]: Since this doesn't benefit users, I think we can omit it.

File: RELEASE.md (hunk 4 above)
Comment (lherman-cs) [reply RELEASE.md:25]: Probably reword this to `High memory allocations in sending audio/video packets`

File: RELEASE.md (hunk 3 above)
Comment (lherman-cs) [reply RELEASE.md:26]: Probably reword this to `Missing logging information about the selected candidates, e.g. IP address, server source, type of candidate chosen and protocol used`.

File: RELEASE.md (hunk 5 above)
Comment (sharadrj) [reply RELEASE.md:9]: Agreed.

File: RELEASE.md (hunk 4 above)
Comment (MushMal) [reply RELEASE.md:25]: I would phrase as "optimized memory allocations during RTP transmission"

File: RELEASE.md (hunk 1 above)
Comment (disa6302) [reply RELEASE.md:10]: Ok. But, the changes would still be part of the release. Do we want to add a note, maybe something like MBedTLS (beta version). ?

File: RELEASE.md (hunk 2 above)
Comment (disa6302) [reply RELEASE.md:15]: Will add once PR #671 is in.

File: RELEASE.md (hunk 1 above)
Comment (lherman-cs) [reply RELEASE.md]: @disa6302 That's a good idea. I like it.
